### PR TITLE
procedure-clairifications

### PR DESCRIPTION
**Pull request by kghamilton89**
Going through doing end-to-end AWS notification testing, I made a few changes to the instructions to help a user understand what they're doing if they choose to go through the "quick" versus "custom" launch.

### CODE REVIEW

File: how-to/aws/cloud-watch-alert/README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Follow the quick [launch procedure](#launch-sandbox-with-automated-slack-and-ema
 
 ### Launch ATSD Sandbox
 
-Follow this procedure to send AWS CloudWatch events into ATSD to enrich standard SNS notifications with additional resource details and AWS console links.
+Follow this procedure to send AWS CloudWatch events into ATSD to enrich standard SNS notifications with additional resource details and AWS console links. Before launching the sandbox, add the desired variables and bind the needed volumes to the sandbox launch command below based on your preferences in [Custom Launch Preferences](#custom-launch-preferences).
```
Comment (rodionos) [inline how-to/aws/cloud-watch-alert/README.md]: This is probably too generic. We need to provide the user a fully functional command to copy and run. How does the use know what are his `desired variables` and why does he need to bind. I think the user should be freed from this level of detail at the beginning of the article.

File: how-to/aws/cloud-watch-alert/README.md [hunk 2]
```diff
@@ -151,11 +149,11 @@ Confirm that your new subscription is active by checking that the **Subscriber**
 
 ![](images/sns-6.png)
 
-ATSD is ready to be configured to notify you via [**Slack Team Messaging**](https://slack.com/) and email.
+ATSD is ready to notify you via [**Slack Team Messaging**](https://slack.com/) and email.
 
 ## Custom Launch Preferences
 
-Customize the generic ATSD launch command with your preferences from these options. After customizing the launch command and executing, return to the [Create SNS Subscription](#create-sns-subscription) section to complete the procedure.
+Customize the generic ATSD launch command with your preferences from these options. After customizing the launch command and executing, return to the [Create SNS Subscription](#create-sns-subscription) section to complete the set-up procedure.
```
Comment (rodionos) [inline how-to/aws/cloud-watch-alert/README.md]: `setup` is an established term. `set-up` is unusual in software manuals.

File: how-to/aws/cloud-watch-alert/README.md (hunk 1 above)
Comment (kghamilton89) [reply how-to/aws/cloud-watch-alert/README.md]: Removed.

File: how-to/aws/cloud-watch-alert/README.md (hunk 2 above)
Comment (kghamilton89) [reply how-to/aws/cloud-watch-alert/README.md]: Removed hyphen.

**Pull request merged**